apiserver/pkg/util/config.go

```diff
@@ -24,6 +24,17 @@ const (
 	RayClusterImageAnnotationKey           = "ray.io/compute-image"
 
 	RayClusterDefaultImageRepository = "rayproject/ray"
+
+	// Max retry times for HTTP Client
+	HTTPClientDefaultMaxRetry = 3
```

Comment:
just FYI (no change requested), the default overall and max timeout reads like your max retry is more proper to be 4.